create_release_branch.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# This creates a new release branch off of production, merges it with main,
```

Comment:
I actually generally just branch off of main to create the release branch, then open the PR against production and it seems to work fine? I think this might have an extra unnecessary step.

Reply:
Yeah you're right.